source/includes/mongodb-compatibility-table-cxx.rst

```diff
@@ -16,94 +16,94 @@
      - MongoDB 3.0
      - MongoDB 2.6
      - MongoDB 2.4
-
    * - 3.6
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-     -
-
+     - 
+     - 
+     - ✓
```

Comment:
3.6 does not have full 5.0 compat support. That is planned in the [3.7 release](https://jira.mongodb.org/projects/CXX?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=unreleased).